jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/parser/DataBodyParser.java

```diff
@@ -26,14 +26,16 @@
 
 public class DataBodyParser extends BodyParser
 {
+    private final RateControl rateControl;
```

Comment:
rateControl is being passed around too many places and set as a field in too many places.  It can just be a filed on HeaderParser.

Reply:
I made this a final field in HeaderParser.